CONTRIBUTING.md

```
@@ -1,202 +1,200 @@
-├── livematch.js           # Socket.IO live multiplayer server (Node.js)
-```
+**Always clean up audio nodes**
+- Creating hover sound nodes without cleanup causes memory leaks
+- Disconnect nodes when done (add cleanup in `useEffect` return or after use)
+- Route gain through shared variables so volume APIs affect the synthetic audio path
 
-## 💡 Contribution Ideas
+### Documentation Coverage
 
-You can help in many ways:
+**Maintain 80%+ docstring coverage**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)